acceptance-tests/src/main/java/it/com/atlassian/bitbucket/jenkins/internal/pageobjects/BitbucketScmWorkflowJob.java

```diff
@@ -31,20 +33,22 @@ public BitbucketScmWorkflowJob(Injector injector, URL url, String name) {
     public BitbucketScmConfig bitbucketScmJenkinsFileSource() {
         select("Pipeline script from SCM");
         select("Bitbucket Server");
+        ExpectedCondition<WebElement> refreshed =
+                ExpectedConditions.refreshed(ExpectedConditions.presenceOfElementLocated(by.id("yui-gen13")));
```

Comment:
this looks potentially flakey? It looks like a generated, what happens if a plugin is updated (since the latest version of all plugins are always installed), I'm guessing this is the best we've got to select on?

Reply:
Just to confirm my understanding, is your concern that the id of the button may change from version to version of the jenkins? Perhaps we could select by something other than the ID?

Reply:
Correct, that is my concern, I'm happy enough with it, thank you for your explanation.